<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsdialog</class>
 <widget class="QDialog" name="settingsdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gidra Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="CloseButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QTabWidget" name="settingsTabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>User</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>511</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="passlable">
            <property name="text">
             <string>New password: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="newPasswordLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="SetNewPassPushButton">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>UI</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>287</width>
           <height>86</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="styleComboBox">
              <item>
               <property name="text">
                <string>Default</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="applyStyleSettingsPushButton">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>711</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>Restart gidra to apply the settings</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Database</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>549</width>
           <height>88</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="exportLabel">
            <property name="text">
             <string>Export:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="exportPathlineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="exportPushButton">
            <property name="text">
             <string>export</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
